server/routes/visitJourney/selectVisitors.ts

```diff
@@ -95,12 +96,30 @@ export default class SelectVisitors {
     const closedVisitPrisoner = visitSessionData.prisoner.restrictions.some(
       restriction => restriction.restrictionType === 'CLOSED',
     )
+    // req.session.visitSessionData.earliestDate = getFunctionCalled()
```

Comment:
Could probably simplify this by something like:
* call  `getBanStatus()` for each visitor to get a `BanStatus[]`
* filter this for where `numDays` is set
* sort this to get the highest value - use this or default (currently 2 days), whichever is greater